core/pom.xml

```diff
@@ -27,10 +27,9 @@
             <groupId>org.glassfish.jersey.inject</groupId>
             <artifactId>jersey-hk2</artifactId>
         </dependency>
-        <!-- Jersey dependency that was available in the JDK before Java 9 -->
```

Comment:
this comment is misleading, it was accidentally copied from the below dependency, so remove it